Modules/HTTP/Sources/HTTP/Middleware/ContentNegotiationMiddleware/ClientContentNegotiatonMiddleware.swift

```diff
@@ -0,0 +1,74 @@
+public struct ClientContentNegotiationMiddleware : Middleware {
```

Comment:
Yeah, I didn't like its length. What do you mean a `Middleware` namespace? A new module?